noaa/gefs/forecast/reformat.py

```diff
@@ -46,20 +48,8 @@
 @sentry_sdk.monitor(
     monitor_slug=f"{template_config.DATASET_ID}-reformat-operational-update",
     monitor_config={
-        "schedule": {"type": "crontab", "value": _CRON_SCHEDULE},
+        "schedule": {"type": "crontab", "value": _OPERATIONAL_CRON_SCHEDULE},
         "timezone": "UTC",
-        # If an expected check-in doesn't come in `checkin_margin`
-        # minutes, it'll be considered missed
-        "checkin_margin": 10,
-        # The check-in is allowed to run for `max_runtime` minutes
-        # before it's considered failed
-        "max_runtime": 120,  # minutes
-        # It'll take `failure_issue_threshold` consecutive failed
-        # check-ins to create an issue
-        "failure_issue_threshold": 1,
-        # It'll take `recovery_threshold` OK check-ins to resolve
-        # an issue
-        "recovery_threshold": 1,
```

Comment:
We can configure the alert parameters in Sentry directly, but I think it's good to have the cron schedule line up with the `CronJob` automatically